optimize.sh

```diff
@@ -45,9 +45,9 @@ for CONTRACTDIR in "$@"; do
     # Build the release for the contract and move it to the artifacts folder
     build_and_move_release() {
       local feature_flag=${1:-}
-      RUSTFLAGS='-C link-arg=-s' cargo build --release --lib --target wasm32-unknown-unknown --locked ${feature_flag}
-      local wasm_output="./target/wasm32-unknown-unknown/release/${pkg_name}".wasm
-      local wasm_name="./target/wasm32-unknown-unknown/release/${pkg_name}${feature_flag:+-}${feature_flag}".wasm
+      RUSTFLAGS='-C link-arg=-s' cargo build --target-dir=/target --release --lib --target wasm32-unknown-unknown --locked ${feature_flag}
+      local wasm_output="/target/wasm32-unknown-unknown/release/${pkg_name}".wasm
+      local wasm_name="/target/wasm32-unknown-unknown/release/${pkg_name}${feature_flag:+-}${feature_flag}".wasm
```

Comment:
Did we remove the `./` for any specific reason?

Reply:
because `PS: The target directory was changed to /target from ./target in the upstream repo.`